src/sign/mod.rs

```diff
@@ -1,10 +1,57 @@
 //! DNSSEC signing.
 //!
 //! **This module is experimental and likely to change significantly.**
+//!
+//! Signatures are at the heart of DNSSEC -- they confirm the authenticity of a
+//! DNS record served by a secure-aware name server.  But name servers are not
```

Comment:
I was taught, though it seems it is not incorrect, not to start sentences with "But". Grammarly advises gently against it too: https://www.grammarly.com/blog/sentences/starting-a-sentence-with-a-conjunction/

Reply:
I was trying to fight back my tendency to write run-on sentences :slightly_smiling_face: I'll rewrite it.